src/compiler/types.ts

```diff
@@ -940,7 +940,8 @@ module ts {
 
     export interface ExportAssignment extends Declaration, ModuleElement {
         isExportEquals?: boolean;
-        expression: Expression;
+        expression?: Expression;
+        type?: TypeNode;

```

Comment:
I did the work to create a new syntax kind, but then felt it was more complexity than value. do you feel strongly about this?